atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -300,8 +300,7 @@ public Void call() throws Exception {
         }
     }
 
-
-    protected static String internalTableName(TableReference tableRef) {
+    public static String internalTableName(TableReference tableRef) {
```

Comment:
This shouldn't be public.  I see that it has to be for this PR to work due to the crosscutting nature of the dbkvs metadata table.  I'm fine with this for now, but we should definitely try to refactor metadata table usage so that we can make the table reference to underlying table string kvs protected.